src/components/Clusters/views/ClusterOverview/ClusterStats.scss

```diff
@@ -33,41 +35,37 @@
       max-height: 100% !important;
     }
   }
+}
 
-  .tall {
-    grid-row: span 2;
-    height: calc(var(--grid-row-height) * 2);
-    grid-column: span 4;
-    width: calc(var(--grid-column-width) * 4);
-  }
+.tall {
```

Comment:
It is a very generic class name and quite specific styles that can't be applied to any object. I would rename it (ex. card-tall) or wrap it in cluster-stats again